### PR TITLE
[Feature] LazyLearn and Graph added

### PR DESCRIPTION
**Pull request by csh3695**
# Major changes
### Tag LazyLearn implemented => LESS DELAY on post create
- `SuggestPending` model created, which contains recently created post. Periodically flushes.
- Added to crontab, time interval set 1min for debugging.
- You can see log at `ml_tag_learn.log`.
### Graph on detail page
- Only post owner can see graph.
- No Graph shown with less than data of 2 days.

<br>  

# Minor changes
Title : 	Post.title
Closed : 	Post.closed
ViewLog : 
date:2019-11-27	view:5
date:2019-11-28	view:6
date:2019-11-29	view:8
date:2019-11-30	view:8
..

### Minor bug fixed
- Null redirect url not accepted bug fixed.

<br>

# Thank You

### CODE REVIEW

**Comment by DHdroid** [comment]
LGTM!

**Comment by digdhg** [comment]
LGTM!

File: backend/aditback/settings.py [hunk 1]
```diff
@@ -137,6 +137,7 @@
 MEDIA_URL = '/media/'
 CRONTAB_COMMAND_SUFFIX = '2>&1'
 CRONJOBS = [
-    ('* * * * *', 'adit.sync.set_data_by_day', '>> ' + BASE_DIR + '/data_update.log')
```
Comment (sanggggg) [inline backend/aditback/settings.py]: set_data_by_day 함수는
* * * * *
가 아니라
0 0 * * * 으로 해야하는것 아닐까요?

File: backend/adit/views.py [hunk 2]
```diff
@@ -283,7 +282,7 @@ def post(self, request):
                         closed=False, thumbnail=thumbnail, open_for_all=open_for_all, view_by_date='')
         adpost.save()
         if adpost.ad_link == "toitself":
-            adpost.ad_link = 'http://localhost:3000/adpost/{}/'.format(str(adpost.id))
```
Comment (sanggggg) [inline backend/adit/views.py]: deploy를 위해서 localhost:3000을 변수로 선언하고 안에 넣어주는 것이 좋을 것 같스비다

**Pull request merged**